scripts/installscripts/buster-install-default.sh

```diff
@@ -1299,8 +1380,10 @@ main() {
         config_mpd
         config_audio_folder "${JUKEBOX_HOME_DIR}"
         config_gpio
+        config_swap
```

Comment:
I think it makes sense to only call this, if the user wants to install Spotify. Could you move the call to the Spotify function?

Reply:
This is no problem to implement - but is it really desirable? Shouldn't we always have some swap created if less than 2 GB is available for the Phoniebox are available since regular playback also requires a bit?

Reply:
What do you think @s-martin ?

Reply:
The swapfile could unnecessary impact the wearlevel and lifetime of the sdcard, if not needed. So maybe make it an optional setting during installation?
For a pi zero or so it might still be helpful?

Reply:
Well - it is optional already. The user is asked if he wants the swapfile to be created and may say 'no'. But then compilation will most certainly fail...

Reply:
Anything else? Or may we merge it?

Reply:
If you add this change we could merge it (sorry for the delay btw. :)

Reply:
Hey, @s-martin . Sorry for not answering this so long.
I do not get what's missing here: The swapfile creation IS already optional in interactive mode. It's only automatically created in non-interactive mode.

In Line 1288, if `$1` is `true`, this triggers the positive action tree, starting in line 1346.
`true` is only passed to the function call if `$INTERACTIVE` is not `true` in Line 1386.
If we are in Interactive mode, the function is called without any parameter in Line 1383 and an interactive prompt asks the user if he/she wants a swapfile to be created or not (Lines 1332:1338).

What else is required here?

Reply:
Done - this should do the trick, I guess.